src/Resources/config/services.xml

```diff
@@ -17,7 +17,6 @@
         </service>
 
         <service id="bitbag_sylius_braintree_plugin.payment_processing.refund" class="BitBag\SyliusBraintreePlugin\PaymentProcessing\RefundPaymentProcessor">
-            <argument type="service" id="session" />
```

Comment:
Why this line has been removed, without any replacement?

Reply:
We don't need replacement. Symfony 5+ provides the session service by default.